From 8f6b8b5f53219ee93c82c2cbdcf436dbf1ed7beb Mon Sep 17 00:00:00 2001
From: Hung Dong <hung.dong.xd@rvc.renesas.com>
Date: Tue, 17 Apr 2018 10:02:54 +0700
Subject: [PATCH 411/504] ARM: dts: iwg21m: Add GPIO interrupt pin of touch LCD
 edt-ft5x06

Refer to "Input: edt-ft5x06 - add GPIO mode config for interrupt pin"
patch for how to use this pin

Signed-off-by: Hung Dong <hung.dong.xd@rvc.renesas.com>
Signed-off-by: vietn <vietn@fsoft.com.vn>
Signed-off-by: Duy Dang <duy.dang.yb@renesas.com>
---
 arch/arm/boot/dts/r8a7742-iwg21m.dts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/boot/dts/r8a7742-iwg21m.dts b/arch/arm/boot/dts/r8a7742-iwg21m.dts
index 7416003..36b9fc7 100644
--- a/arch/arm/boot/dts/r8a7742-iwg21m.dts
+++ b/arch/arm/boot/dts/r8a7742-iwg21m.dts
@@ -350,6 +350,7 @@
 		interrupt-parent = <&gpio0>;
 		interrupts = <24 IRQ_TYPE_EDGE_FALLING>;
 		reset-gpios = <&gpio1 29 GPIO_ACTIVE_LOW>;
+		int-gpios = <&gpio0 24 GPIO_ACTIVE_LOW>;
 	};
 
 	codec: sgtl5000@a {
-- 
2.7.4

